docs/integrations/saas-cloud/box.md

```diff
@@ -80,153 +80,326 @@ _sourceCategory=box  type "event_type" login
+On the Collection page, the [Health](/docs/manage/Health-Events#Collection_page) and Status for Sources is displayed. Use [Health Events](/docs/manage/Health-Events) to investigate issues with collection.<br/><img src={useBaseUrl('img/send-data/box-source3.png')} alt="Box" />
+
+You can click the text in the Health column, such as **Error**, to open the issue in Health Events to investigate.<br/><img src={useBaseUrl('img/send-data/hover-c2c-error.png')} alt="Box" />
+
+Hover your mouse over the status icon to view a tooltip with details on the detected issue.<br/><img src={useBaseUrl('img/send-data/health-error-generic.png')} alt="Box" />
+
+
+### Create a Box Source
+
+When you create a Box Source, you add it to a Hosted Collector. Before creating the Source, identify the Hosted Collector you want to use or create a new Hosted Collector. For instructions, see [Create a Hosted Collector](/docs/send-data/Hosted-Collectors#Create_a_Hosted_Collector).
+
+To configure a Box Source:
+1. In Sumo Logic, navigate to **Manage Data** > **Collection** and open the **Collection** tab.
+2. On the Collectors page, click **Add Source** next to a Hosted Collector.
+3. Select **Box**.<br/><img src={useBaseUrl('img/send-data/box-source2.png')} alt="Box" />
+4. Enter a **Name** for the Source. The **description** is optional. <br/><img src={useBaseUrl('img/send-data/box-source1.png')} alt="Box" />
+5. (Optional) For **Source Category**, enter any string to tag the output collected from the Source. Category [metadata](/docs/search/Get-Started-with-Search/Search-Basics/Built-in-Metadata) is stored in a searchable field called `_sourceCategory`.
+6. **Forward to SIEM**. Check the checkbox to forward your data to [Cloud SIEM Enterprise](/docs/cse).
+
+When configured with the **Forward to SIEM** option the following metadata fields are set:
+
+<table>
+  <tr>
+   <td>
+Field Name
+   </td>
+   <td>Value
+   </td>
+  </tr>
+  <tr>
+   <td><code>_siemVendor</code>
+   </td>
+   <td>Box
+   </td>
+  </tr>
+  <tr>
+   <td><code>_siemProduct</code>
+   </td>
+   <td>Enterprise Events
+   </td>
+  </tr>
+  <tr>
+   <td><code>_siemFormat</code>
+   </td>
+   <td>JSON
+   </td>
+  </tr>
+  <tr>
+   <td><code>_siemEventID</code>
+   </td>
+   <td>&#123;event_type&#125;
+   </td>
+  </tr>
+</table>
+
+
+1. (Optional) **Fields**. Click the **+Add** link to add custom log metadata [Fields](/docs/manage/fields).
+    * Define the fields you want to associate, each field needs a name (key) and value.
+        * A green circle with a checkmark is shown when the field exists and is enabled in the Fields table schema.
+        * An orange triangle with an exclamation point is shown when the field doesn't exist, or is disabled, in the Fields table schema. In this case, an option to automatically add or enable the nonexistent fields to the Fields table schema is provided. If a field is sent to Sumo that does not exist in the Fields schema or is disabled it is ignored, known as dropped.
```

Comment:
Green circle & orange triangle thumbnails needed here. You can refer to most Integrations or Source docs.

Reply:
done

Reply:
I don't see them -- be sure to push the commit to this PR once you've made the change locally.